commands/commandeer.go

```diff
@@ -360,6 +362,13 @@ func (c *commandeer) loadConfig(mustHaveConfigFile, running bool) error {
 		if c.destinationFs != nil {
 			// Need to reuse the destination on server rebuilds.
 			fs.Destination = c.destinationFs
+		} else if createMemFs && c.renderStaticFilesToDisk {
+			// Writes the dynamic output on memory,
+			// while serve others directly from publishDir
+			publishDir := config.GetString("publishDir")
+			writableFs := afero.NewBasePathFs(afero.NewMemMapFs(), publishDir)
+			publicFs := afero.NewOsFs()
+			fs.Destination = afero.NewCopyOnWriteFs(afero.NewReadOnlyFs(publicFs), writableFs)
```

Comment:
Also re my comment below, I don't think you can change this global `fs.Destination`.

Reply:
This is changeable and, actually,  required.
Without this, we cannot access any static files (,or files in `public` directory) via local server.
We can see it almost identical to "afero.MemMapFs" (like the below clause).
The difference is that the fs can serve files from `public` directory as well as memory